api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java

```diff
@@ -206,5 +206,10 @@ public TracerProvider getTracerProvider() {
     public ContextPropagators getPropagators() {
       return delegate.getPropagators();
     }
+
+    @Override
+    public TracerBuilder tracerBuilder(String instrumentationName) {
+      return delegate.tracerBuilder(instrumentationName);
+    }
```

Comment:
do I completely not understand how default methods and method dispatch works or something? I don't see why this is necessary given the default implementation.

Reply:
Here, the default implementation actually does something sensible:

https://github.com/open-telemetry/opentelemetry-java/blob/d9e1151336afc5778a110863bb9aa4178af567e3/api/all/src/main/java/io/opentelemetry/api/trace/TracerProvider.java#L75-L77

Still, we bypass any override that the delegate might have, calling `delegate.getTracerProvider().tracerBuilder(instrumentationName)` instead of `delegate.tracerBuilder(instrumentationName)`. This will hopefully do the same, but this bypass would still be very surprising in case one adds logging to `tracerBuilder` or an auto-instrumenting agent instruments it, etc.

Reply:
I assume this is also due to the missing implementation in the ObfuscatedTracerProvider. I'd rather fix it there, rather than patch the issue and not fix the underlying problem.

Reply:
ugh. I also missed that this was the implementation in the obfuscated impl. Sorry for the noise.